README.md

```diff
@@ -2,19 +2,57 @@
 A dockless mobility data API built with Python/Sanic
 
 ## Installation
-** Requires Python 3.6+ **
+
+### About the "Database"
+
+The source database for the API is a modified geojson file. TODO: add sample data and link to processing tools to generate source data.
+
+### Option 1: Run w/ Python 3
 
 1. Clone repo and `cd` into it.
+
 2. Install python requirements:
 ```python
 pip install -r requirements.txt
 ```
-3. Start the server:
+3. Install [libspatialindex](http://libspatialindex.github.io/)
+
+4. Copy `grid.json` source data to `../dockless-api/app/data`
+
+5. Start the server:
 ```python
 python app/app.py
 ```
 
-4. Make a request:
+6. Make a request:
 ```shell
 curl http://localhost:8000/v1/trips?xy=-97.75094341278084,30.276185988411257&flow=destination
 ```
+
+### Option 2: Run w/ Docker
+
+1. Install [docker](https://www.docker.com/) and start the engine:
+`systemctl start docker`
```

Comment:
I actually think it's more straightforward to install Docker for Desktop.
https://www.docker.com/products/docker-desktop